screens/Tutorials/TutorialScreen1.js

```diff
@@ -0,0 +1,116 @@
+import React, { useState } from "react";
+import { Image, StyleSheet, TouchableHighlight, View } from "react-native";
+import KitText from "../../components/KitText";
+import Colors from "../../constants/Colors";
+import KitButtonSupreme from "../../components/KitButtonSupreme";
+import CompletionBar from "../../components/challenges/CompletionBar";
+
+export default function Tutorial1Screen(props) {
+  const { navigation } = props;
+  const kitglobeImage = require("../../assets/images/kitglobe.png");
+
+  return (
+    <View style={styles.container}>
+      <TouchableHighlight 
+      onPress = {() => {navigation.navigate("Tutorial2")}}
+      underlayColor='none'
+      >
+        <View style={styles.card}>
+        <View style={styles.textContainer}>
+          <KitText
+            style={styles.text}
+            color={Colors.KIT_BLACK}
+            size={30}
+            fontWeight="bold"
+          >
+            Welcome to kit!
+          </KitText>
+          <KitText
+            style={styles.text} 
+            color={Colors.KIT_BLACK}
+            size={16}
+          >
+            An app to foster long distance friendships.
+          </KitText>
+        </View>
+        <View style={styles.imageContainer}>
+        <Image source={kitglobeImage}/>
+        </View>
+        <View style={styles.challengeContainer}>
+        <CompletionBar numCompleted={1} numInTeam={4} mainColor={"#569684"}/>
```

Comment:
Whereever we are using the CompletionBar component, can we add a comment that states this isn't what the design actually specifies but we're going with what we have at the moment:
Something like:
`TODO: refactor into actual design later`